synapse/lib/types.py

```diff
@@ -155,6 +156,14 @@ def cmpr(valu):
             return valu in norms
         return cmpr
 
+    def _ctorCmprRange(self, vals):
+        minv = self.norm(vals[0])[0]
```

Comment:
This has a undefined behavior that gets a bit weird
```
cli> storm inet:ipv4 +inet:ipv4*range=(1, 2, 3)

inet:ipv4=0.0.0.1
        .created = 2018/12/03 19:58:46.515
        :asn = 0
        :loc = ??
        :type = private
inet:ipv4=0.0.0.2
        .created = 2018/12/03 19:58:46.523
        :asn = 0
        :loc = ??
        :type = private
complete. 2 nodes in 7 ms (285/sec).
```

Since you allow N values to come into the `vals` list, we can throw IndexErrors if only one valu is present; and only eat the first two values if N > 2.  `Type.indxByRange` has a pair of precondition checks to ensure that it is used consistently.  We should include those checks here.

Reply:
```
cli> storm inet:ipv4 +inet:ipv4*range=(1,)

('err', ('IndexError', {'efile': 'types.py', 'eline': 161, 'mesg': 'list index out of range'}))
complete. 0 nodes in 2 ms (0/sec).
``` likewis with a single valu

Reply:
When these preconditions are added, ensure that there are tests which cause them the raise exceptions.